### PR TITLE
fix: TrackStateType printout in wrong order

### PR DESCRIPTION
**Pull request by timadye**
`operator<<(std::ostream& os, ConstTrackStateType t)` prints out the flag bits as a binary number, with bit 0 (`MeasurementFlag`) at the end. This is belied by the label `MPOHMS`, which should be `SMHOPM`.

This PR fixes the label. Another option would be to reverse the order of bit printout. Please comment if you think that would be better.

### CODE REVIEW

**Comment by timadye** [comment]
This was found while investigating #2905

**Review by andiwand — APPROVED** [review]
lgtm

File: Core/include/Acts/EventData/TrackStateType.hpp [hunk 1]
```diff
@@ -115,8 +115,8 @@ class ConstTrackStateType {
     for (std::size_t i = 0; i < TrackStateFlag::NumTrackStateFlags; i++) {
       trunc[i] = bs[i];
     }
-    // MeasurementParameterOutlierHoleMaterialSharedhit
-    os << "MPOHMS=" << trunc;
+    // SharedhitMaterialHoleOutlierParameterMeasurement
```
Comment (paulgessinger) [inline Core/include/Acts/EventData/TrackStateType.hpp:118]: Ok I see.
Should this print the literal byte content, or should it print the ordering in the enum though?

Comment (timadye) [reply Core/include/Acts/EventData/TrackStateType.hpp:118]: I'd be happy to do it the other way. This way seemed the simplest unless someone had an opinion one way or the other.

Comment (andiwand) [reply Core/include/Acts/EventData/TrackStateType.hpp:118]: I have no opinion on the order as long as it is consistent 😄 anyways if somebody sees a problem we can always change it

Comment (paulgessinger) [reply Core/include/Acts/EventData/TrackStateType.hpp:118]: Let's merge it as is.

**Pull request merged**